Turn/src/Game.cpp

```diff
@@ -52,6 +52,20 @@ string Game::InitializePlayerName() {
 	return name;
 }
 
+char Game::InitializePlayerGender() {
+    ClearScreen();
+    char gender;
+    do {
+        cout << "What is your gender (M or F)?"
+            << endl << endl
+            << "> ";
+
+        cin >> gender;
```

Comment:
I suggest that `gender` is converted to uppercase so that both upper/lower case m/f works

Reply:
Also, ClearScreen() should be called after every iteration in the while loop